cpp/oneapi/dal/detail/serialization.hpp

```diff
@@ -0,0 +1,104 @@
+/*******************************************************************************
```

Comment:
Why don't you move it using `git mv`? Now it is depicted as brand new file

Reply:
This is brand new file